exercises/concept/lucas-numbers/.docs/instructions.md

```diff
@@ -0,0 +1,64 @@
+# Instructions
```

Comment:
I like the setup. My only remark is that the introduction (before the first task) does not actually define the sequence's rules. These are defined in the individual task, but I wonder if perhaps it would be nice to define the sequence in the intro too.